azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/service/project/AzureProjectService.java

```diff
@@ -45,19 +54,37 @@ public AzureProjectService(AzureHttpService azureHttpService) {
 
     public AzureArrayResponseModel<TeamProjectReferenceResponseModel> getProjects(String organizationName) throws HttpServiceException {
         String requestSpec = API_SPEC_ORGANIZATION_PROJECTS
-                                 .defineReplacement("{organization}", organizationName)
+                                 .defineReplacement(PATH_ORGANIZATION_REPLACEMENT, organizationName)
                                  .populateSpec();
         Type responseType = new TypeToken<AzureArrayResponseModel<TeamProjectReferenceResponseModel>>() {}.getType();
         return azureHttpService.get(requestSpec, responseType);
     }
 
     public TeamProjectResponseModel getProject(String organizationName, String projectId) throws HttpServiceException {
         String requestSpec = API_SPEC_ORGANIZATION_PROJECTS_INDIVIDUAL
-                                 .defineReplacement("{organization}", organizationName)
-                                 .defineReplacement("{projectId}", projectId)
+                                 .defineReplacement(PATH_ORGANIZATION_REPLACEMENT, organizationName)
+                                 .defineReplacement(PATH_PROJECT_ID_REPLACEMENT, projectId)
                                  .populateSpec();
         Type responseType = new TypeToken<TeamProjectResponseModel>() {}.getType();
         return azureHttpService.get(requestSpec, responseType);
     }
 
+    public AzureArrayResponseModel<ProjectPropertyResponseModel> getProjectProperties(String organizationName, String projectId) throws HttpServiceException {
+        String requestSpec = API_SPEC_ORGANIZATION_PROJECT_PROPERTIES
+                                 .defineReplacement(PATH_ORGANIZATION_REPLACEMENT, organizationName)
+                                 .defineReplacement(PATH_PROJECT_ID_REPLACEMENT, projectId)
+                                 .populateSpec();
+        requestSpec = String.format("%s?%s=%s", requestSpec, AzureHttpService.AZURE_API_VERSION_QUERY_PARAM_NAME, "5.1-preview.1");
```

Comment:
I think the third variable should be removed here as well or replaced with a static variable